lightkurve/tests/test_mast.py

```diff
@@ -93,12 +91,12 @@ def test_kepler_tpf_from_archive():
     assert(isinstance(tpfs, list))
     assert(isinstance(tpfs[0], KeplerTargetPixelFile))
     assert(tpfs[0].quarter == 1)
-    # If we ask for a nearby target, it should only give back one extra with the same quarter.
+    # Ask for one extra nearby target
     tpfs = KeplerTargetPixelFile.from_archive(
-        5728079, cadence='long', radius=60, quarter=1, targetlimit=2)
+        "GJ 9827", cadence='long', radius=60, campaign=12, targetlimit=2)
     assert(isinstance(tpfs, list))
     assert(isinstance(tpfs[0], KeplerTargetPixelFile))
-    assert(tpfs[0].quarter == tpfs[1].quarter)
+    assert(tpfs[0].campaign == tpfs[1].campaign)
     assert(tpfs[0].keplerid != tpfs[1].keplerid)
 
```

Comment:
Why did this unit test change? It looks like just the target has changed not purpose of the test?